WordPress/Sniffs/Theme/NoAutoGenerateSniff.php

```diff
@@ -77,18 +77,22 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
 		}
 
 		foreach ( $this->generated_themes as $generated_theme ) {
+			/*
+			 * The check can have false positives like artisteers but chances of that happening
+			 * in a valid theme is low.
+			 */
 			// The check can have false positives like artisteers but chances of that in a valid theme is low.
```

Comment:
Thanks for catching that. I cherry picked this from another branch I was working on and that clearly doesn't always work as well as it should.